addons/web/static/src/legacy/js/views/kanban/kanban_column.js

```diff
@@ -208,7 +208,10 @@ var KanbanColumn = Widget.extend({
         }
         this.trigger_up('close_quick_create'); // close other quick create widgets
         var context = this.data.getContext();
-        context['default_' + this.groupedBy] = viewUtils.getGroupValue(this.data, this.groupedBy);
+        // Since a date field can have a granularity in a groupby (date_field:granularity), and viewUtils.getGroupValue requires
+        // only the field name, we have to split out the eventual granularity
+        var groupedBy = (Array.isArray(this.groupedBy)) ? this.groupedBy[0].split(':')[0] : this.groupedBy.split(':')[0];
+        context['default_' + groupedBy] = viewUtils.getGroupValue(this.data, groupedBy);
```

Comment:
same remark as in kanban_controller.
Side note: should there be a viewUtils for this?